solc/CommandLineInterface.cpp

```diff
@@ -353,23 +353,23 @@ void CommandLineInterface::handleSignatureHashes(std::string const& _contract)
 	if (!m_options.compiler.outputs.signatureHashes)
 		return;
 
-	Json::Value interfaceSymbols = m_compiler->interfaceSymbols(_contract);
+	Json interfaceSymbols = m_compiler->interfaceSymbols(_contract);
 	std::string out = "Function signatures:\n";
-	for (auto const& name: interfaceSymbols["methods"].getMemberNames())
-		out += interfaceSymbols["methods"][name].asString() + ": " + name + "\n";
+	for (auto const& [name, _]: interfaceSymbols["methods"].items())
+		out += interfaceSymbols["methods"][name].get<std::string>() + ": " + name + "\n";
```

Comment:
```suggestion
	for (auto const& [name, value]: interfaceSymbols["methods"].items())
		out += value.get<std::string>() + ": " + name + "\n";
```
I saw a few minor ones like these and ignored them, since in the end it doesn't matter, but this one looks too weird to refrain from commenting :-).